pkg/asset/installconfig/ovirt/credentials.go

```diff
@@ -1,92 +1,272 @@
 package ovirt
 
 import (
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/asn1"
 	"fmt"
-	"net/url"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strconv"
+	"strings"
 
+	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 	"gopkg.in/AlecAivazis/survey.v1"
 )
 
-func askCredentials() (Config, error) {
-	c := Config{}
+// clientHTTP struct - Hold info about http calls
+type clientHTTP struct {
+	saveFilePath string // Path for saving file (GET method)
+	urlAddr      string // URL or Address
+	skipVerify   bool   // skipt cert validatin in the http call
+	certPool     *x509.CertPool
+}
+
+// EngineConfig struct - Hold all info about user environment
+var EngineConfig = Config{}
+
+// HTTPResource struct - Hold info for managing http calls
+var HTTPResource = clientHTTP{}
+
+// Import PEM into the System Pool
+func (c clientHTTP) importCertIntoSystemPool(pemFilePath string) bool {
+	c.certPool, _ = x509.SystemCertPool()
+	if c.certPool == nil {
+		logrus.Debug("Failed to load cert pool.... Creating new cert pool")
+		c.certPool = x509.NewCertPool()
+		return false
+	}
+	for _, rawSubject := range c.certPool.Subjects() {
+		var rdnSequence pkix.RDNSequence
+		_, err := asn1.Unmarshal(rawSubject, &rdnSequence)
+		if err != nil {
+			logrus.Debug("Could not unmarshal der formatted subject")
+			return false
+		}
+		var name pkix.Name
+		name.FillFromRDNSequence(&rdnSequence)
+		if strings.Contains(name.CommonName, EngineConfig.FQDN) {
+			logrus.Debug("Found FQDN in the cert list of subjects! CommonName: ", name.CommonName)
+			return true
+		}
```

Comment:
The trust bundle is not required to have the common names for the APIs, they are used to sign the server certificates. So this check is incorrect.